C/ConnectFourSolver/build_tarballs.jl

```diff
@@ -2,42 +2,35 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+julia_version = v"1.5.3"
+
 name = "ConnectFourSolver"
 version = v"0.1.0"
 
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/findmyway/connect4.git", "db9b13d14951d35e4f3f21e4f2ecaa93a906f277"),
-    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/armv7l/1.3/julia-1.3.1-linux-armv7l.tar.gz", "965c8fab2214f8ce1b3d449d088561a6de61be42543b48c3bbadaed5b02bf824"; unpack_target="julia-arm-linux-gnueabihf"),
-    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz", "faa707c8343780a6fe5eaf13490355e8190acf8e2c189b9e7ecbddb0fa2643ad"; unpack_target="julia-x86_64-linux-gnu"),
-    ArchiveSource("https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-x86_64-apple-darwin14.tar.gz", "f2e5359f03314656c06e2a0a28a497f62e78f027dbe7f5155a5710b4914439b1"; unpack_target="julia-x86_64-apple-darwin14"),
-    ArchiveSource("https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-x86_64-w64-mingw32.tar.gz", "c7b2db68156150d0e882e98e39269301d7bf56660f4fc2e38ed2734a7a8d1551"; unpack_target="julia-x86_64-w64-mingw32")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-case "$target" in
-	arm-linux-gnueabihf|x86_64-linux-gnu)
-        Julia_PREFIX=${WORKSPACE}/srcdir/julia-$target/julia-1.3.1
-        ;;
-    x86_64-apple-darwin14|x86_64-w64-mingw32)
-        Julia_PREFIX=${WORKSPACE}/srcdir/julia-$target/juliabin
-        ;;
-esac
-
-mkdir connect4/build
-cd connect4/build
-cmake -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DJulia_PREFIX=$Julia_PREFIX -DCMAKE_PREFIX_PATH=$prefix/destdir/lib/cmake/JlCxx -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_FIND_ROOT_PATH=${prefix} ..
+cd connect4*
+install_license LICENSE
+mkdir build
+cd build
+cmake \
+    -DCMAKE_FIND_ROOT_PATH=${prefix} \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DJulia_PREFIX=${prefix} \
+    ..
```

Comment:
I took the liberty of sorting these arguments alphabetically and generally aligning them with other JLLs, in the hope that this will help a bit with future maintenance.